src/shared/components/entry/SequenceView.tsx

```diff
@@ -100,21 +101,29 @@ export const SequenceInfo = ({
   ];
 
   return (
-    <Sequence
-      sequence={dataToDisplay?.value}
-      onShowSequence={() => setIsoformToFetch(isoformId)}
-      infoData={infoData}
-      accession={isoformId}
-      downloadUrl={apiUrls.sequenceFasta(isoformId)}
-      onBlastClick={() =>
-        history.push(LocationToPath[Location.Blast], {
-          parameters: { sequence: dataToDisplay?.value },
-        })
+    <LazyComponent
+      fallback={
+        <div style={{ wordBreak: 'break-all' }}>
```

Comment:
Is there a benefit to inline styles in this instance?

Reply:
I don't think so. We're already loading a stylesheet with this page so I've added a classname for this instead of an inline style